lib/importers/rating_importer.rb

```diff
@@ -55,7 +55,8 @@ def self.update_ratings(article_groups)
   ####################
   def self.update_article_ratings(articles, ratings)
     articles.each do |article|
-      article.rating = ratings[article.title]
+      article.rating_all = ratings[article.title]
```

Comment:
Instead of saving the original full-spectrum rating separately, I think we should only store that rating (as `article.rating`) and then basically do what you are doing with default_rating() below within the ArticleHelper methods 'rating_display' and 'rating_priority'

I don't want to add a new column to the database for this.